src/Groups/types.jl

```diff
@@ -77,6 +75,7 @@ TODO: document this
 abstract type GAPGroup <: AbstractAlgebra.Group end
 abstract type GAPGroupElem{T<:GAPGroup} <: AbstractAlgebra.GroupElem end
 
+abstract type GAPGroupElem{T<:GAPGroup} <: AbstractAlgebra.GroupElem end
```

Comment:
```suggestion
```
I think this line is redundant (it is identical to the one above?). Perhaps this slipped in during a rebase?

Reply:
Yes, probably.